### PR TITLE
Payee 196 fe library upgrade check previous tax years

### PR DESCRIPTION
**Pull request by Jamie22Butler**
- [x] Unit tests passing
- [ ] Coverage reached
- [ ] Acceptance tests passing
- [ ] Reviewed
- [ ] Squash

### CODE REVIEW

**Review by vivekhmrc — APPROVED** [review]
LGTM 👍

File: app/views/paye/HistoricPayAsYouEarnView.scala.html [hunk 1]
```diff
@@ -23,7 +23,7 @@
 @import includes.link
 @import controllers.auth.AuthenticatedRequest
 
-@this(mainTemplate: MainTemplate)
+@this(mainTemplate: MainTemplate, main: MainTempleteNew)
```
Comment (brendanstraughan) [inline app/views/paye/HistoricPayAsYouEarnView.scala.html:26]: Should the old main template injection be removed?

File: app/views/IncomeTaxSummaryView.scala.html [hunk 2]
```diff
@@ -45,26 +45,38 @@
 
 @header = @{
     ptaHeader(
-        displayBackLink = true,
+        displayBackLink = false,
```
[the remark this answers is not in the document]
Comment (Jamie22Butler) [reply app/views/IncomeTaxSummaryView.scala.html]: Yes, new injection as:

` customBackLinkContent = Some(backLinkToChooseTaxYear)`

**Review by brendanstraughan — APPROVED** [review]

**Review by masahmad — APPROVED** [review]
LGTM

File: app/views/paye/HistoricPayAsYouEarnView.scala.html [hunk 3]
```diff
@@ -47,23 +47,31 @@
...
+        showGdsVersion = true)
 }
 
-@mainTemplate(
+@main(
     title = messages("tai.paye.heading", TaxPeriodLabelService.taxPeriodLabel(viewModel.taxYear.year)),
-    articleClasses = Some("full-width"),
     authedUser = Some(request.taiUser),
-    pageHeader = Some(header),
+    pageTitle = Some("Check your Income Tax"),
```
Comment (sergi-hmrc) [inline app/views/paye/HistoricPayAsYouEarnView.scala.html:58]: shouldn't this come from messages?

Comment (Jamie22Butler) [reply app/views/paye/HistoricPayAsYouEarnView.scala.html:58]: [](url
Screen Shot 2022-04-21 at 15 50 23
)

**Pull request merged**